mlflow/store/artifact/artifact_repo.py

```diff
@@ -189,7 +190,8 @@ def _download_file(src_artifact_path, dst_local_dir_path):
 
         # Wait for downloads to complete and collect failures
         failed_downloads = {}
-        for f in as_completed(futures):
+        for f in (pbar := tqdm(as_completed(futures), total=len(futures))):
+            pbar.set_description(f"Downloading artifact {futures[f]}")
```

Comment:
```suggestion
        pbar = tqdm(as_completed(futures), total=len(futures))
        for f in pbar:
            pbar.set_description(f"Downloading artifact {futures[f]}")
```

I'd use a simple assignment here.